test/functional/tests/lazy_writes/test_flush_huge_dirty_data.py

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+import pytest
+from datetime import timedelta
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode, CacheModeTrait, CleaningPolicy, SeqCutOffPolicy
+from api.cas.cli import stop_cmd
+from core.test_run import TestRun
+from storage_devices.disk import DiskType, DiskTypeLowerThan, DiskTypeSet
+from test_tools.disk_utils import Filesystem
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import IoEngine, ReadWrite
+from test_tools.fs_utils import remove
+from test_utils.filesystem.file import File
+from test_utils.os_utils import sync, Udev
+from test_utils.size import Size, Unit
+
+file_size = Size(420, Unit.GiB)
```

Comment:
I assume that in original test filesize was ~600GB, because some day there was a problem with such size, so I'd like you to increase this value in this test as well.

Reply:
You're right, previously I probably looked at the description, which was out of date. I'll do what You ask for.